benchmarks/templates/benchmarks/domain-information.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'benchmarks/base.html' %}
+{% block content %}
+
+    {% if user %}
+        <section id="profile" class="container center">
+            <section id = profile-info>
+                <h1>{{ user.email }}</h1>
+
+                {% if "language" in request.path %}
+                    <form action="{% url 'display-name-language' %}" method="post">
+                        {% csrf_token %}
+                        <label class="label">Display name</label>
+                        <div class="field has-addons">
+                            <p class="control">
+                                <input class="input" type="text" name="display_name" value="{{ user.display_name }}">
+                            </p>
+
+                            <p class="control">
+                                <input type="submit" value="Save" class="button">
+                            </p>
+                        </div>
+                    </form>
+                {% else %}
+                    <form action="{% url 'display-name-vision' %}" method="post">
+                        {% csrf_token %}
+                        <label class="label">Display name</label>
+                        <div class="field has-addons">
+                            <p class="control">
+                                <input class="input" type="text" name="display_name" value="{{ user.display_name }}">
+                            </p>
+
+                            <p class="control">
+                                <input type="submit" value="Save" class="button">
+                            </p>
+                        </div>
+                    </form>
+                {% endif %}
```

Comment:
redundant and not scalable. Do away with these `if-else` distinctions and make it variable dependent.